testcafe/newResource.js

```diff
@@ -7,23 +7,63 @@ fixture`Add New Resource`
   .page(NewResourcePage.url());
 
 test('Add new resource, basic', async t => {
-  const newName = 'New Resource Name';
-
-  // TODO: These are all currently required, but should they be?
-  const newAddress = {
-    address1: '123 Fake St.',
-    city: 'San Francisco',
-    stateOrProvince: 'CA',
-    country: 'United States',
-    postalCode: '94110',
+  // TODO: Currently, all that is required is a name and an address
+  // what should the minimum data be for a new resource?
+  const data = {
+    name: 'New Resource Name',
+    address: {
+      address1: '123 Fake St.',
+      city: 'San Francisco',
+      stateOrProvince: 'CA',
+      country: 'United States',
+      postalCode: '94110',
+    },
+    phone: {
+      number: '1234567890',
+      type: 'LandLine',
+    },
+    website: 'https://theBestResource.best',
+    email: 'theBestResource1337@aol.com',
+    description: 'A description of this resource',
+    note: 'A new note about this resource',
   };
 
-  await t.typeText(newResourcePage.name, newName, { replace: true });
-  await Object.keys(newAddress).reduce(
-    (_t, prop) => _t.typeText(newResourcePage.address[prop], newAddress[prop], { replace: true }),
+  // Add name
+  await t.typeText(newResourcePage.name, data.name, { replace: true });
+
+  // Add address
+  await Object.keys(data.address).reduce(
+    (_t, prop) => _t.typeText(
+      newResourcePage.address[prop],
+      data.address[prop],
+      { replace: true },
+    ),
     t,
   );
 
+  // Add phone
+  await t.click(newResourcePage.addPhoneButton);
+  const phone = NewResourcePage.getPhone(-1);
+  await t
+    .typeText(phone.number, data.phone.number, { replace: true })
+    .typeText(phone.serviceType, data.phone.type, { replace: true });
+
+  // Add website
+  await t.typeText(newResourcePage.website, data.website, { replace: true });
+
+  // Add email
+  // For some reason, testCafe puts the email in backwards
+  const reversedEmail = data.email.split('').reverse().join('');
```

Comment:
Are you sure about this? It looks like you're failing the Travis CI test because it's being input backwards:

```
AssertionError: expected 'moc.loa@7331ecruoseRtseBeht' to deeply
      equal 'theBestResource1337@aol.com'
```

Reply:
Well, it fails locally without this, but fails Travis CI with it, but Travis CI is more important.
Fixed.

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/12485238/64488883-2e4fd300-d20a-11e9-912c-50847de80160.png">